cpp/environment/AngularSeparation.cc

```diff
@@ -15,28 +15,19 @@ using namespace tbb;
 
 namespace freud { namespace environment {
 
-AngularSeparation::AngularSeparation() : m_n_query_points(0), m_n_points(0),
-    m_n_global(0), m_n_equiv_orientations(0), m_tot_num_neigh(0) {}
+AngularSeparation::AngularSeparation() {}
 
 AngularSeparation::~AngularSeparation() {}
 
+//! Clip v if it is outside the range [lo, hi].
+float clamp(float v, float lo, float hi) {
+    return std::max(lo, std::min(v, hi));
+}
+
```

Comment:
Assuming that this was copied from my recent bug-fixes to Steinhardt, there are probably more places that we'd want to use this. I'd recommend moving this logic into some common header and making it accessible everywhere.

Reply:
I created #448 to address this separately.